docs/zh/standards/srgf.md

```diff
@@ -124,7 +124,9 @@ UIGF 4.0 **不具备对旧有标准的向下兼容性**，且与旧有标准的
               "1",
               "2",
               "11",
-              "12"
+              "12",
+              "21",
```

Comment:
SRGF has already been deprecated. You should not continue to update the deprecated standard.